### PR TITLE
update doc in functional.py

### PR DESCRIPTION
**Pull request by Jiangtang**
explicitly specify inputs for two functions: binary_cross_entropy and binary_cross_entropy_with_logits:
binary_cross_entropy takes logistic sigmoid values as inputs
binary_cross_entropy_with_logits takes logits as inputs

Fixes #{issue number}

### CODE REVIEW

**Comment by mruberry** [comment]
Thanks for your interest, @Jiangtang, but this PR is too small (see [here](https://github.com/pytorch/pytorch/wiki/Developer-FAQ#my-pr-was-rejected-for-being-too-small--what-gives)). If you're interested in getting started with PyTorch development then I suggest looking for open issues where the team has commented we would accept a PR fixing them.

The [contribution guide](https://pytorch.org/docs/stable/community/contribution_guide.html) might also be interesting.
